test/src/main/java/org/apache/accumulo/test/upgrade/ConfigPropertyUpgraderIT.java

```diff
@@ -62,13 +65,35 @@ public class ConfigPropertyUpgraderIT {
   private static ZooKeeper zooKeeper;
   private static ZooReaderWriter zrw;
 
+  // Create legacy renamer for this test
+  private static final DeprecatedPropertyUtil.PropertyRenamer MASTER_MANAGER_RENAMER =
+      DeprecatedPropertyUtil.PropertyRenamer.renamePrefix("master.",
+          Property.MANAGER_PREFIX.getKey());
```

Comment:
Instead of keeping that specific renamer around, can we create a dummy renamer for testing purposes, so it's not tied to any specific previous code that looks like it tech debt?

Reply:
Yeah I will rework this a bit in combination with my other comment so it's a bit cleaner and uses a dummy renamer.